ivre/active/data.py

```diff
@@ -417,6 +418,17 @@ def merge_scanner_scripts(
         nports = len(ports["ports"])
         res["ports"][proto]["count"] = nports
         res["ports"]["count"] = res["ports"].get("count", 0) + nports
+        # If the list of scanned ports is almost exactly the same as
+        # the list of ports scanned by default by a scanner, assume it
+        # is.
+        for (dports, scanner_name) in DEFAULT_SCANNED_PORTS.get(proto, []):
```

Comment:
```suggestion
        for (dports, scanner_name, scanner_info) in DEFAULT_SCANNED_PORTS.get(proto, []):
```

Reply:
Why did you remove this change? Was it on purpose or a side effect of the rebase?